### DISCUSSION TITLE
每个固件里都有个名称，每个有什么区别

### DISCUSSION THREAD

**Discussion by getsources** [ophub/amlogic-s9xxx-armbian#2498] [Other] [answered]
![Screenshot_2024-08-21-11-57-16-591_com.github.android-edit.jpg](https://github.com/user-attachments/assets/d94ecc7f-93af-4fee-8699-5f5ab685e39e)

**Comment by ophub** [comment, marked as answer]
不同的版本，ubuntu系列的是jammy和noble，debian系列的是bullseye和bookworm

**Reply by fanfan089** [reply]
大佬能具体点吗，比如功能上有什么区别

**Comment by ophub** [comment]
这个区别可以网上搜一下介绍
